composer.json

```diff
@@ -19,7 +19,8 @@
         "dniccum/phone-number": "^1.1",
         "inspheric/nova-email-field": "^1.42",
         "tipoff/authorization": "^2.5.5",
-        "tipoff/support": "^1.7.2"
+        "tipoff/support": "^1.7.2",
+        "livewire/livewire": "^2.0"
```

Comment:
@pdbreen We are going to be using Livewire in the majority of the packages. Should we add this dependency to tipoff/support?

Reply:
I'd say if you think most packages will be using it, then sure.  If its just a handful that need it, probably better to leave it as a package pull in.